src/main/java/com/isroot/stash/plugin/YaccServiceImpl.java

```diff
@@ -139,13 +139,29 @@ private boolean isCommitExcluded(Settings settings, YaccCommit commit) {
         return errors;
     }
 
+    private List<YaccError> checkCommiterEmailRegex(Settings settings, YaccCommit commit) {
+        List<YaccError> errors = Lists.newArrayList();
+        String regex = settings.getString("commiterEmailRegex");
+        //String regex = "^.*\\@"+emailDomain+"$";
```

Comment:
Can you also remove this since this commented code isn't really doing anything. Probably left over from testing?